chillkang/charmhub.io
# WD-9313 Add a mermaid diagram component to render mermaid diagrams on the interface detail page

## Done
- Instead of using plugins, use a mermaid diagram component to render mermaid diagrams on the interface detail page.

## How to QA
- Check out this branch locally
- Go to /kafka/integrations
- Click `Kafka_client` and see if a mermaid diagram is rendered

## Issue / Card
Fixes #https://warthogs.atlassian.net/browse/WD-9313

## Screenshots
<img width="1265" alt="Screenshot 2024-03-25 at 7 44 17 AM" src="https://github.com/chillkang/charmhub.io/assets/90341644/7ce8f378-6cd7-4cb8-8ba5-0cc6f32ad6bd">
